### DISCUSSION TITLE
How to round values

### DISCUSSION THREAD

**Discussion by techwavedev** [grafana/grafana#38058] [Alerting]
Hi,

how I'm able to round a value in annotation 
for example  my {{ $values }} returns:  map[B:20.114754098360656 C:0]

I would like to round the B in this case for any decimal places 

Cheers,

**Comment by mtischle** [comment]
Hi,

I'm looking for the same. My understanding from here:
https://grafana.com/docs/grafana/latest/alerting/unified-alerting/alerting-rules/alert-annotation-label/
is, that {{ template "humanize" $values.B }} should do the trick. But it doesn't work. I get "template \"humanize\" not defined" in the logs (on 8.3.0).

Thanks,

**Reply by fulopbencus** [reply]
`{{ humanize (index $values "B").Value }}`

**Comment by nourspace** [comment]
Try this
```
{{ printf "%.0f" $values.B.Value }}
```

**Reply by StarXsw** [reply]
it works, thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by StarXsw** [comment]
您的邮件我已收到，谢谢

**Comment by StarXsw** [comment]
您的邮件我已收到，谢谢
